Examples/snippets.lua

```diff
@@ -133,6 +133,14 @@ local function bash(_, command)
 	return res
 end
 
+-- Returns a snippet_node wrapped around an insert_node whose initial
+-- text value is set to the current date in the desired format.
+local date_input = function(args, state, fmt)
+    local fmt = fmt or "%Y-%m-%d"
+    local f = io.popen("date +'" .. fmt .. "'", "r")
```

Comment:
Prefer  `os.date`  so it's os independent.

Reply:
Done.